### DISCUSSION TITLE
Unable to Verify

### DISCUSSION THREAD

**Discussion by komi786** [community/community#125546] [GitHub Education]
### Hi everyone,

Hello,
I applied for GitHub copilot but application was rejected first time. I would like to apply again with more proof but it does not move to next option and stuck as "select school"  with following message. `Hi,  You were last verified as a student on Apr 8, 2024. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.`. Please help me here

**Comment by Ygilany** [comment]
I have the same issue with my Faculty account reverification as well
